### PR TITLE
Add weather screen

### PR DESCRIPTION
**Pull request by ErylKenner**

### CODE REVIEW

**Review by michaelmartinelli15 — APPROVED** [review]

File: App.tsx [hunk 1]
```diff
@@ -26,7 +27,7 @@ export default function App() {
         </Tab>
         <Tab heading="Weather">
           <View style={{ flex: 1, margin: 10 }}>
-            <Text>Welcome to Weather!</Text>
+            <WeatherScreen source="Sacremento" destination="Reno" />
```
Comment (kylebrain) [inline App.tsx:30]: You misspelled Sacramento

File: components/WeatherScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            <Icon name="ios-search" />
+            <Input placeholder="Add Location" />
+          </Item>
+        </Header>
+      </View>
+
+      
+      <ScrollView scrollEnabled={true} contentContainerStyle={{flexGrow: 1}} style={{height: "10000px", overflow: "scroll"}}>
+        <WeatherCard name="Truckee Meadows" updatedTime="5:05pm" temperature={57} useCelcius={false} cloudiness="Mostly cloudy" rainingChance="25% chance of rain"/>
+        <WeatherCard name="Truckee Meadows" updatedTime="5:05pm" temperature={57} useCelcius={false} cloudiness="Mostly cloudy" rainingChance="25% chance of rain"/>
```
Comment (kylebrain) [inline components/WeatherScreen.tsx:40]: Change to a different place

**Review by kylebrain — CHANGES_REQUESTED** [review]
Coupla thangs

**Review by kylebrain — APPROVED** [review]
LGTM

**Review by michaelmartinelli15 — APPROVED** [review]

**Pull request merged**